### DISCUSSION TITLE
How long do a Pull Request live?

### DISCUSSION THREAD

**Discussion by mattiasmaahl** [Brewtarget/brewtarget#577] [General]
So was going through the Pull Requests and the the more elderly ones do have more than one issue to resolve before merging.
Can we close them or do they live forever?
This also includes the one I made with the Debian dependencies which feels a bit out dated due to the recent activity around building and github actions.

This is more of a "keeping things tidy" kind of thing, so let me know how you think we should handle this kind of thing.
I'm fine with letting them be there, it's just not likely that they will ever be merged.

**Comment by mikfire** [comment]
If possible, I would like to get them merged. It may take some effort on our part but it seems a little rude to discard somebody else's work.

If the change can't be merged, then close it.

**Comment by mattiasmaahl** [comment]
You make a good point. 
I will look into if The original author want to help or if I can make it fit in tje current source for the old pull requests after I'm done with the MSI windows installer convedsion. I will have a draft ready today, but will not likely have it all done. Most of the work is to make the installer find and copy the old database and modify to registry entries.

**Comment by matty0ung** [comment]
We now only have one open "old" pull request.  At some point I'll have a look at whether/how we can merge it to the newer code base.  But, meanwhile, archiving this discussion.
